experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -337,6 +376,16 @@ function is_tensor_power(V::LieAlgebraModule{C}) where {C<:RingElement}
   return get_attribute(V, :type, :fallback) == :tensor_power
 end
 
+function base_module(V::LieAlgebraModule{C}) where {C<:RingElement}
+  @req is_dual(V) || is_exterior_power(V) || is_symmetric_power(V) || is_tensor_power(V) "Not a power module."
+  return get_attribute(V, :base_module)::LieAlgebraModule{C}
+end
+
+function base_modules(V::LieAlgebraModule{C}) where {C<:RingElement}
+  @req is_direct_sum(V) "Not a direct sum module."
+  return get_attribute(V, :base_modules)::Vector{LieAlgebraModule{C}}
```

Comment:
Here, I changed it from an abstract `Tuple{Vararg{LieAlgebraModule{C}}}` to a concrete `Vector{LieAlgebraModule{C}}`. @fingolfin